src/models/polar/constraints.jl

```diff
@@ -207,6 +207,34 @@ function flow_constraints(polar::PolarForm, cons, buffer)
     wait(ev)
     return
 end
+
+function flow_constraints_grad(polar::PolarForm, cons, buffer, reduction::Function=sum)
+    nlines = PS.get(polar.network, PS.NumberOfLines())
+    f = polar.topology.f_buses
+    t = polar.topology.t_buses
+    fr_flow = similar(cons)
+    to_flow = similar(cons)
+    fr_vmag = buffer.vmag[f]
+    to_vmag = buffer.vmag[t]
+    Δθ = buffer.vang[f] .- buffer.vang[t]
+    cosθ = cos.(Δθ)
+    sinθ = sin.(Δθ)
+    function lumping(vmag, vang)
+        fr_vmag = vmag[f]
+        to_vmag = vmag[t]
+        Δθ =vang[f] .- vang[t]
```

Comment:
just for linting :angel: 
could we add a space before `=`?

Reply:
Yea, well this has changed a little now 😆